ethereum/api/src/main/java/org/hyperledger/besu/ethereum/api/jsonrpc/internal/methods/engine/EngineNewPayloadV4.java

```diff
@@ -78,9 +78,6 @@ protected ValidationResult<RpcErrorType> validateParameters(
       return ValidationResult.invalid(
           RpcErrorType.INVALID_PARENT_BEACON_BLOCK_ROOT_PARAMS,
           "Missing parent beacon block root field");
-    } else if (maybeRequestsParam.isEmpty()) {
```

Comment:
I think we still need this check as we need to have execution requests, but they can be an empty list

Reply:
Done